app/_src/gateway/reference/cli.md

```diff
@@ -314,8 +319,10 @@ Options:
 
  -p,--prefix      (optional string)   Override prefix directory.
 
+{% if_version lte:3.9.x %}
  --v              verbose
  --vv             debug
+{% endif_version %}
```

Comment:
```suggestion
 --v              verbose
 --vv             debug
```
Why would this be removed? I checked the CLI in 3.10.0.0-rc.2, both of these flags are still here.

Reply:
weird, I didn't get them in the output. I'll add them back